core/domain/prod_validation_jobs_one_off.py

```diff
@@ -202,6 +202,10 @@ def _get_external_id_relationships(cls, item):
             dict(str, (ndb.Model, list(str)). A dictionary whose keys are
             field names of the model to validate, and whose values are tuples
             that consist of the external model class and list of keys to fetch.
+
+        Raises:
+            NotImplementedError: Method or function hasn't
+                been implemented yet.
```

Comment:
Can we move this line above? (or part of it*)

[Ditto elsewhere]

Reply:
done

Reply:
I don't see this resolved, can you please check!